tests/suites/dashboard/dashboard.sh

```diff
@@ -6,8 +6,7 @@ run_dashboard_deploy() {
 	juju expose dashboard
 	juju relate dashboard controller
 
-	juju wait-for application dashboard
-	sleep 5 # short wait for relation data to update
+  wait_for "controller" "$(active_condition "dashboard")"
```

Comment:
Run `shfmt -w` on this file.

Reply:
Yes sorry, just saw the static analysis failure.